monitor-stddev.py

```
@@ -0,0 +1,724 @@
+            adjusted_latencies.extend([current_probe_adjusted_latency] * probe_interval)
+            response_status_codes.extend([current_probe_reresponse_status_code] * probe_interval)
+
+            cur_sample_size: int = len(errors)
+            if cur_sample_size > SAMPLE_SIZE_SECONDS:
+
+                analysis_errors = errors[-SAMPLE_SIZE_SECONDS:]
+                analysis_eff_latencies = effective_latencies[-SAMPLE_SIZE_SECONDS:]
+                analysis_adj_latencies = adjusted_latencies[-SAMPLE_SIZE_SECONDS:]
+                analysis_status_codes = response_status_codes[-SAMPLE_SIZE_SECONDS:]
+
+                valid_eff_latencies = [l for l in analysis_eff_latencies if l is not None]
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)